kotlin/ktor/resources/application.conf

```diff
@@ -0,0 +1,9 @@
+ktor {
+    deployment {
+        port = 3000
+        port = ${?PORT}
```

Comment:
Not a duplicate ?

Reply:
It allows to override the port if a environment variable PORT is present.

Reply:
ok